infra/aws/org-bootstrap.md

```diff
@@ -287,7 +287,7 @@ Follow the same steps you took to bootstrap the `org-root` account to
 bootstrap your `id` account. Once you've done that, you can set up the
 users and groups that team members will use to access all other AWS
 resources. Truss has made two modules which will help you get this done,
-[`iam-cross-acct-src`](https://registry.terraform.io/modules/trussworks/iam-cross-acct-src/aws) and
+[`terraform-layout-example`](https://github.com/trussworks/terraform-layout-example) and
```

Comment:
*Technically*, terraform-layout-example is not a Terraform module. Maybe instead, "Truss uses the iam-user-group and iam-cross-acct-dest modules to do this; see terraform-layout-example for how this is done."

Reply:
I appreciate your attention to detail and basically always find your comments helpful. I'm going to change these because you are correct and this is definitely an improvement.

In the future, would you mind just using the `nit: ` prefix for comments like this? It goes down a little easier for me than hearing you say you're sorry & call yourself pedantic. Own it, sis.

Reply:
I'll try. :) Thank you for understanding. :)